lib/rules/jsx-handler-names.js

```diff
@@ -24,23 +24,42 @@ module.exports = {
       type: 'object',
       properties: {
         eventHandlerPrefix: {
-          type: 'string'
+          type: ['string', 'boolean']
```

Comment:
can this be made to prevent use of `true`? perhaps `oneOf: [{ type: 'string' }, false]` or similar?

Reply:
Done